eng/targets/Imports.targets

```diff
@@ -169,7 +169,10 @@
       Disable analyzers via an MSBuild property settable on the command line.
     -->
     <ItemGroup>
-      <Analyzer Remove="@(Analyzer)" />
+      <!--
+        Roslyn.Diagnostics.Analyzers provides diagnostic suppressors that are required for clean builds.
+      -->
+      <Analyzer Remove="@(Analyzer)" Condition="'%(FileName)' != 'Roslyn.Diagnostics.Analyzers'" />
```

Comment:
I am not sure this is desirable - this means we will end up running analyzers everywhere, which will likely takeaway lot of local build time improvements. A better approach might be to have a separate package `Roslyn.Diagnostics.Suppressors` which should only have suppressors for compiler diagnostics, and apply this workaround for it. Additionally, please make sure that you measure the local build time regression from this change.